### ISSUE TITLE
Only getting a 404?

### ISSUE THREAD

**Issue by OmgImAlexis** [SwitchbladeBot/invidget#49]
I had no issues with this until today when I noticed my link now shows a 404. The invite still works.

```
https://invidget.switchblade.xyz/nhS4f9PJpg?2
```

**Comment by jnsp-vsc** [comment]
Yes just got the same, image not loading and getting a error message with "404 page not found" when visiting https://invidget.switchblade.xyz/Secret_server_invite
